File: ts/wam/5788.ts
export const WamNativeContactsNuxEvent = 5788 as const

export interface NativeContactsNuxEventEvent {
  /** field 1, wire `native_contacts_nux_event_type` */
  nativeContactsNuxEventType?: NATIVECONTACTSNUXEVENTTYPE
  /** field 3, wire `native_contacts_nux_entry_point` */
  nativeContactsNuxEntryPoint?: NATIVECONTACTSNUXENTRYPOINT
}

File: go/wam/5788.go
const WamNativeContactsNuxEvent = 5788 // channel: regular

type NativeContactsNuxEventEvent struct {
	NativeContactsNuxEventType NATIVECONTACTSNUXEVENTTYPE `wam:"native_contacts_nux_event_type"` // field 1
	NativeContactsNuxEntryPoint NATIVECONTACTSNUXENTRYPOINT `wam:"native_contacts_nux_entry_point"` // field 3
}